### ISSUE TITLE
Questions about the person-specific expressions

### ISSUE THREAD

**Issue by RenYurui** [JustusThies/NeuralVoicePuppetry#16]
Thanks for the great work.
I am a little bit confused about the person-specific expressions in the paper. 
I have read the code and understand the operation of this step. My questions are:
1. What do the person-specific expressions learn? something like the specific talking style? (as the same pronunciation may lead to different expressions)
2. Have you try to directly learn the expression coefficients from the audio? what the difference?
3. If we have several videos of one person with different talking styles is it better to learn video-specific expressions?
Thanks

**Comment by JustusThies** [comment]
Hi RenYurui,

Thanks for the questions!

The person-specific expressions are in our case video-specific (we assume that we only have a single video of the target). The person-specific expression base is used to model person specific expressions that vary in the form and also strength (note that the expression vectors are displacement vectors that are applied to the neutral face). Having an additional conditioning to control the expression style of a specific person (depending on the mood of the person) is an interesting research direction.

In fact, I also tried to directly learn from the audio spectograms, the problem is that it overfits to the relatively small dataset that we used. In contrast, the audio-features of the deepspeech network are trained on a much larger dataset.

Best,
Justus
